src/metabase/models/serialization/base.clj

```diff
@@ -373,14 +374,18 @@
   (fn [model _ _] model))
 
 (defmethod load-update! :default [model-name ingested local]
-  (let [model (db/resolve-model (symbol model-name))
-        pk    (models/primary-key model)
-        id    (get local pk)]
+  (let [model    (db/resolve-model (symbol model-name))
+        pk       (models/primary-key model)
+        id       (get local pk)
+        adjusted (if (-> model models/properties :timestamped?)
+                   (assoc ingested :updated_at (mi/now))
+                   ingested)]
     (log/tracef "Upserting %s %d: old %s new %s" model-name id (pr-str local) (pr-str ingested))
-    ; Using the two-argument form of [[db/update!]] that takes the model and a HoneySQL form for the actual update.
-    ; It works differently from the more typical `(db/update! 'Model id updates...)` form: this form doesn't run any of
-    ; the pre-update magic, it just updates the database directly.
-    (db/update! (symbol model-name) {:where [:= pk id] :set ingested})
+    ;; Using the two-argument form of [[db/update!]] that takes the model and a HoneySQL form for the actual update.
+    ;; It works differently from the more typical `(db/update! 'Model id updates...)` form: this form doesn't run any of
+    ;; the pre-update magic, it just updates the database directly.
+    ;; Therefore we manually set the :updated_at time.
+    (db/update! model {:where [:= pk id] :set adjusted})
```

Comment:
On that note, **why** wouldn't we want to use the "normal" version of Toucan `update!` here? It seems like we really do want that `pre-update`, type transformation, and properties logic to happen for us. I think it looks really fragile having SerDes have to know about and duplicate all of the Toucan logic -- super error-prone in the future if we add more stuff to `metabase.models` and people don't know they need to update SerDes as well